android/app/src/main/java/net/minetest/minetest/GameActivity.java

```diff
@@ -180,11 +181,17 @@ public void openURI(String uri) {
 	}
 
 	public String getUserDataPath() {
-		return Utils.getUserDataDirectory(this).getAbsolutePath();
+		return Objects.requireNonNull(
```

Comment:
please move the non-null check info `getUserDataDirectory`

Reply:
I rearranged some codes regarding non-null-ness.